lib/index.js

```diff
@@ -163,7 +163,8 @@ internals.tryToRequire = (path) => {
         return require(path);
     }
     catch (err) {
-        Hoek.assert(err.code === 'MODULE_NOT_FOUND', err);
+        // Must be an error specifically from trying to require the passed (normalized) path
+        Hoek.assert(err.code === 'MODULE_NOT_FOUND' && ~err.message.indexOf(path), err);
```

Comment:
Ah, sorry about that!  Glad you like it.  I probably wouldn't have used it on a project, but given that this is my own lil module, I guess I took a liberty :P

At the same time, if we all know what it is then we can start using it ;)

Thanks for the review!!